wikipedia-dark.user.css

```diff
@@ -4192,6 +4194,26 @@ regexp("https?:\/\/([\w\-]{2,}\.m\.(wiki(pedia|books|news|quote|source|versity|v
   tr[bgcolor], td[bgcolor] {
     background-color: var(--gray-28);
   }
+  /* popups fixes */
+  .mwe-popups .mwe-popups-container {
+    background-color: var(--gray-2) !important;
+  }
+  .mwe-popups .mwe-popups-container .mwe-popups-discreet {
+    background-color: var(--white) !important;
+  }
+  .mwe-popups .mwe-popups-container a {
+    color: var(--gray-c) !important;
+  }
+  .mwe-popups-extract::after {
+    background-image: linear-gradient(to right, transparent, var(--gray-2) 50%) !important;
+  }
+  .mw-mmv-ttf-ellipsis::before {
+    background-image: linear-gradient(to right, transparent, var(--gray-2)) !important;
+  }
+  .mwe-popups-extract .mwe-popups-fade,
+  .mw-mmv-permission-box .mw-mmv-permission-text .mw-mmv-permission-text-fader {
+    background-image: linear-gradient(to bottom, transparent, var(--gray-2)) !important;
+  }
```

Comment:
I would suggest minimize the use of !important unless its absolutely necessary for the override.

Also both popups are missing a border/box shadow where the arrow side of the popup.

Reply:
For this part of the code, I actually just copy and pasted it from the desktop version. I notice that the desktop version also have the same issue. So maybe for this PR, I include only these changes to fix white background, and perhaps when I had time I would look into the missing border and create separate PR for it

Reply:
Sure but it is already an issue on desktop version, just because something has existed for a long time, it shouldnt be perpetuated if not absolutely necessary.

Reply:
So after testing, just the above vendor-provided selector is necessary for that last rule, and `!important` can be removed from all of them. Everything else is good.